### DISCUSSION TITLE
n8n Integration

### DISCUSSION THREAD

**Discussion by AnalogJ** [twentyhq/twenty#4118] [Ideas]
## Scope & Context

Since you already have an integration for Zapier, it would be nice to add one for n8n as well.

**Comment by FelixMalfait** [comment]
Agree it would be great! It's not going to be prioritized on the near-term roadmap for the core team but if someone in the community worked on it we'd be happy to support :)

**Comment by wgicio** [comment]
+1 for this n8n is awesome

**Comment by gabrielbryk** [comment]
Also would love to see n8n integration, found this community node for it but not sure how functional it is
https://github.com/shodgson/n8n-nodes-twenty
